src/libraries/Common/tests/System/Net/Http/Http3LoopbackStream.cs

```diff
@@ -199,6 +211,17 @@ static int EncodeHttpInteger(long longToEncode, Span<byte> buffer)
             }
         }
 
+        public async Task<int> ReadDataStreamAsync(byte[] payload)
```

Comment:
@CarnaViire  I tried to keep an uniformity in the tests so they look like the others, also because this file had some preparing functionality(to send different bytes of memory, and I thought there fitted best my header preparing). 
I will move them inside the Http3LoopbackConnection code since I think they will fit better there and switch to quic streams entirely

Reply:
What exactly do you want to move inside Http3LoopbackConnection?

Reply:
@CarnaViire  I thought more about it and I do not think that I should move something in the `Http3LoopbackConnection`, but I need to keep the `SendWTStreamHeaderAsync` function, so I will just add it as a function inside the tests

Reply:
Fixed